Documentation/LargeProjects.md

```diff
@@ -0,0 +1,143 @@
+
+Note that the beginning parts of this are taken from the default github Unity .gitignore with
+MRTK-specific additions later. Also note that some of these rules assume certain paths, and
+can be modified based on where MRTK is located within your project.
+
+```
+# This .gitignore file should be placed at the root of your Unity project directory
+#
+# Get latest from https://github.com/github/gitignore/blob/master/Unity.gitignore
+#
+/[Ll]ibrary/
+/[Tt]emp/
+/[Oo]bj/
+/[Bb]uild/
+/[Bb]uilds/
+/[Ll]ogs/
+
+# MemoryCaptures can get excessive in size.
+# They also could contain extremely sensitive data
+/[Mm]emoryCaptures/
+
+# Asset meta data should only be ignored when the corresponding asset is also ignored
+!/[Aa]ssets/**/*.meta
+
+# Uncomment this line if you wish to ignore the asset store tools plugin
+# /[Aa]ssets/AssetStoreTools*
+
+# Autogenerated Jetbrains Rider plugin
+/[Aa]ssets/Plugins/Editor/JetBrains*
+
+# Visual Studio cache directory
+.vs/
+
+# Gradle cache directory
+.gradle/
+
+# Autogenerated VS/MD/Consulo solution and project files
+ExportedObj/
+.consulo/
+*.csproj
+*.unityproj
+*.sln
+*.suo
+*.tmp
+*.user
+*.userprefs
+*.pidb
+*.booproj
+*.svd
+*.pdb
+*.mdb
+*.opendb
+*.VC.db
+
+# Unity3D generated meta files
+*.pidb.meta
+*.pdb.meta
+*.mdb.meta
+
+# Unity3D generated file on crash reports
+sysinfo.txt
+
+# Builds
+*.apk
+*.unitypackage
+
+# Crashlytics generated file
+crashlytics-build.properties
+
+# NuGet Build Process Generated
+# This section covers files that are generated when consuming/using MRTK from NuGet
+# May not be relevant if consuming via .unitypackages
+/NuGet/*
+!/NuGet/BuildSource.proj
+/MRTKBuild/
+/MSBuild/
+/PackagesCopy/
+!/scripts/Packaging/NuGetRestoreProject.csproj
+
+# MSBuildForUnity generated
+# This section covers files that are generated by the MSBuildForUnity system
+# which MRTK uses to consume certain NuGet packages. Useful for when MSBuildForUnity
+# is enabled on the project (in the MRTK configurator)
+MSBuildForUnity.Common.props
+/*.msb4u.csproj.meta
+*.msb4u.csproj.meta
+/*.msb4u.sln.meta
+*.msb4u.sln.meta
+Dependencies*
+Nuget.config*
+
+# DotNetAdapater specific
+# This section covers files that are deployed by the imported DotNetWinRT NuGet package
+# This is primarily used for Unity Editor to HoloLens 2 in-editor remoting 
+!/Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/DotNetAdapter/DotNetAdapter.csproj
+Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/DotNetAdapter/.bin/
+Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/DotNetAdapter/.obj/
+```
+
+## ProjectPreferences.asset file
+
+Project-wide MRTK settings can be found under the Edit -> Project Settings
+-> Mixed Reality Toolkit location. These settings will be saved to a file
+at this location in the Assets folder:
+
+```
+Assets/MixedRealityToolkit.Generated/ProjectPreferences.asset
+```
+
+If your project will have multiple collaborators, it is recommended that
```

Comment:
We should try to avoid the use of you/your [Documentation Guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#writing-style)